### DISCUSSION TITLE
Vehicle minimum occupancy

### DISCUSSION THREAD

**Discussion by kazitanvirahsan** [google/or-tools#2854] [Routing (and legacy CP) questions] [answered]
Hai guys,

I really hope that someone can able to help with the following scenario.

In our solution, almost every load gets delivered but we notice some vehicles (Not all) use only 20%-30% of their capacity before delivery which is not a realistic solution. For example, 

Vehicle capacity 24

Load 1      (Pickup)                    3
Load 2      (Pickup)                    8

Load 1      (Delivery)                  3
Load 2      (Delivery)                  8

Load 3      (Pickup)                    7
Load 4      (Pickup)                    9

Load 3      (Delivery)                  7
Load 4      (Delivery)                  9

Is there any way we can force vehicles with 80%-90% capacity before starting to delivery as below

Load 1      (Pickup)                    3
Load 2      (Pickup)                    8
Load 3      (Pickup)                    1
Load 4      (Pickup)                    2

Load 1      (Delivery)                  3
Load 2      (Delivery)                  8
Load 3      (Delivery)                  1
Load 4      (Delivery)                  2

How can we make sure Trucks always have minimum occurrence or go empty?

Kazi

**Comment by lperron** [comment, marked as answer]
duplicate.
